### ISSUE TITLE
Deadswaps - a solution to death

### ISSUE THREAD

**Issue by Alonzo-Coeus** [decentraland/proposals#46]
i purpose a way to avoid the problem of people loosing interest in decentraland and leaving there plots. if we kept our current system spawn would die the first people would getplots die or move on. This leaves us with a problem of dead plots with no activity and we would have a expanding bubble of graveyardness (that is now a word).

The solution i suggest is adding a new transaction the "dead-swap" with this a plot of land which has not been updated within the last <insert time>, can be taken over by anyone but to avoid the destruction of land the land is moved so the old owner gets some land from the swapper and the swapper gets the new land he wants.

using this system you don't get spawn death you get a flow of active players toward the center or hubs of high value land. you also get a inverse flow of inactive land going to the far reaches of dland or the low cost areas.

One of the problems with the above system is some people don't want to change there tile so another tx type is added called PROMPT or PING this acts to renew the lands lease so it does not get moved.

in conclusion we have created a system which keeps higher cost and popular areas active by treating the land system like domain names you "own" your land but you must renew it to stop it from being swapped for a active piece of land.

Edit:
With dland switching to a smart contract based system we have to change the system  a bit. Instead of switching plots the plot is set to unclaimed, again so after 1 year the plot cleared and built on.

**Comment by Alonzo-Coeus** [comment]
Originaly here :: https://forum.decentraland.org/topic/11/dead-swaps/4
